Documentation/ceph-filesystem-crd.md

```diff
@@ -31,7 +31,8 @@ spec:
     replicated:
       size: 3
   dataPools:
-    - failureDomain: host
+    - name: replicated-pool
```

Comment:
pool in the name seems redundant, how about removing the pool suffix from the examples?
```suggestion
    - name: replicated
```

Reply:
but since it is for example, i just wanted to mention *pool* suffix for storage class. Otherwise, in storage class we will have pool name just as myfs-replicated. Having myfs-replicated-pool (with pool name replicated-pool) a bit more user friendly, haven't it?

Reply:
In the storage class, the setting is: `pool: myfs-replicated-pool`, right? If the setting name is `pool`, isn't it clear it's a pool?

Reply:
Okay, you win :)